yarn-project/end-to-end/src/cli_docs_sandbox.test.ts

```diff
@@ -26,10 +26,24 @@ describe('CLI docs sandbox', () => {
   }, 60_000);
 
   const waitForSandboxWithCli = async () => {
+    const docs = `
+// docs:start:node-info
+% aztec-cli get-node-info
+Node Info:
+
+Sandbox Version: 0.7.7
+Compatible Nargo Version: 0.11.1-aztec.0
+Chain Id: 31337
+Protocol Version: 1
+Rollup Address: 0x0dcd1bf9a1b36ce34237eeafef220932846bcd82
```

Comment:
Shouldn't we assert that the returned text from the command matches this? It's a bit annoying since we need to load the versions from the release-please-manifest and the noir-version jsons. Or we can just parse it manually, and use an `expect.anything()` for those values.